File: advertlibrary/src/main/java/com/huimee/advertlibrary/AdvertImageView.java
package com.huimee.advertlibrary;

import android.app.Activity;
import android.content.Context;
import android.util.AttributeSet;
import android.widget.ImageView;

import androidx.appcompat.widget.AppCompatImageView;

/**
 * Created by YX on 2019/12/14 10:40.
 */
public class AdvertImageView extends AppCompatImageView {
    public AdvertImageView(Context context) {
        super(context);
    }

    public AdvertImageView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public AdvertImageView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public  void setImage(Activity activity,  String s) {
        DialogUtils.INSTANCE.showDialog(activity,s,3,-1,this);
    }
}

File: advertlibrary/src/main/java/com/huimee/advertlibrary/AdvertImageView.kt
package com.huimee.advertlibrary

import android.app.Activity
import android.content.Context
import android.util.AttributeSet
import android.widget.ImageView


/**
 *   Created by YX on 2019/12/11 15:26.
 */
class AdvertImageView @JvmOverloads constructor(
    context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
) : ImageView(context, attrs, defStyleAttr) {


    fun setImage(activity: Activity, s: String) {
        DialogUtils.showDialog(activity,s,3,-1,this)
    }


}